### PR TITLE
PKG Bump scikit-learn to 1.1.0

### LINKED ISSUE #2571: scikit-learn / 1.1.0
## 🐍 Package Update

- Package Name and Version : scikit-learn / 1.1.0
- Package URL : https://files.pythonhosted.org/packages/8b/99/b1ec652f2d60a13871a3053f312f9d78977be57e420f2a49d52ba503f1f4/scikit-learn-1.1.0.tar.gz
- Package Dependencies that needs to be resolved first: numpy, scipy, joblib, threadpoolctl

### PR DESCRIPTION
**Pull request by ryanking13**
Close #2571

### CODE REVIEW

**Comment by hoodmane** [comment]
Thanks @ryanking13!

I am hoping to switch to keeping the patches always in the format produced by `git format-patch` and consumed by `git am` and always rebasing them whenever we update regardless of whether it is needed. I think this will ensure that patches stay well documented and updates are as easy as possible when actual conflicts occur rebasing the patch. This also has the benefit that ```sh
git checkout old-tag
git am /path/to/pyodide/packages/some_pkg/patches/*
git rebase old-tag --onto=new-tag
git format-patch new-tag -o /path/to/pyodide/packages/some_pkg/patches
```
automatically preserves the patch comments and authors.

**Comment by ryanking13** [comment]
Yeah, I'm +1 for it.

File: packages/scikit-learn/meta.yaml [hunk 1]
```diff
@@ -7,8 +7,8 @@ source:
   sha256: 80f9904f5b1356adfc32406725dd94c8cc9c8d265047d98390033a6c238cbb29
 
   patches:
-    - patches/disable-openmp.patch
     - patches/0001-Cythonize-_cython_blas.pyx-with-include-path-pointin.patch
+    - patches/0002-Disable-omp.patch
```
Comment (hoodmane) [inline packages/scikit-learn/meta.yaml:11]: Oops

**Comment by hoodmane** [comment]
Thanks @ryanking13!

**Pull request merged**